src/Illuminate/Cache/Repository.php

```diff
@@ -481,18 +481,18 @@ public function offsetUnset($key)
     /**
      * Calculate the number of minutes with the given duration.
      *
-     * @param  \DateTime|int  $duration
-     * @return int|null
+     * @param  \DateTime|float  $duration
+     * @return float|null
      */
     protected function getMinutes($duration)
     {
         if ($duration instanceof DateTime) {
-            $fromNow = Carbon::now()->diffInMinutes(Carbon::instance($duration), false);
+            $fromNow = Carbon::now()->diffInSeconds(Carbon::instance($duration), false) / 60;
```

Comment:
I'm a little confused here because these look equal? Isn't diffInMinutes the exact same as diffInSeconds / 60?